### DISCUSSION TITLE
Multilevel correlation vs. linear mixed model

### DISCUSSION THREAD

**Discussion by pedfig** [easystats/correlation#239] [Q&A]
I was trying to compute the multilevel correlations using the package and a linear mixed model. Still, I don’t seem to be able to calculate precisely the values provided by the correlation package. Can you give me information on why this is the case? Below is an example. Thank you!

```

data %>% select(ID, var1, var2) %>%
    correlation(partial = FALSE, multilevel = TRUE, p_adjust = "none")

Parameter1 | Parameter2 |     r |             CI | t(70) |      p
-----------------------------------------------------------------
var1             |      var2     | -0.29 | [-0.49, -0.07] | -2.57 | 0.012*


example <- lmerTest::lmer(var1 ~ var2 + (1|ID), data = data)

parameters::model_parameters(example, effects = "fixed", ci_method = "satterthwaite")

Parameter   | Coefficient |    SE |           95% CI |     t |    df |      p
-----------------------------------------------------------------------------
(Intercept) |      424.12 | 10.58 | [401.75, 446.48] | 40.09 | 16.54 | < .001
var2       |       -9.43 |  3.45 | [-16.32,  -2.54] | -2.74 | 61.34 | 0.008 

t_to_r(-2.74, df_error = 61.34)

r     |         95% CI
----------------------
-0.33 | [-0.52, -0.09]

```

**Comment by DominiqueMakowski** [comment]
Sorry I missed that.

As you can see, the operation is taken care of here:

https://github.com/easystats/correlation/blob/a5444ca5a706c08002eb652c1294aa5650f2197c/R/cor_test.R#L151-L154

which calls [adjust](https://github.com/easystats/datawizard/blob/master/R/adjust.R) with multilevel set to TRUE. That happens in `cor_test()`. However in `correlation()` there's also this step:

https://github.com/easystats/correlation/blob/a5444ca5a706c08002eb652c1294aa5650f2197c/R/correlation.R#L392

Maybe that's the cause? To be honest I'm not sure, one would need to reproduce the different steps manually

**Reply by pedfig** [reply]
Thank you!
